backend/contracts/ICoffeeProfile.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @description Interface used to ensure all new coffees following a concrete pattern.
```

Comment:
Just wondering if there is any way to share contracts between this and the client to not have to upkeep two?

Reply:
yeah, I felt dirty doing that as well. We could use something like this https://yarnpkg.com/en/docs/workspaces to make a package that just exports out common contracts. Created #27 for this.

Reply:
Sounds fancy. I like it.